lib/code_climate/test_reporter/formatter.rb

```diff
@@ -39,11 +39,11 @@ def format(result)
       # actually private ...
       def short_filename(filename)
         return filename unless ::SimpleCov.root
-        filename = filename.gsub(/^#{::SimpleCov.root}/, ".").gsub(/^\.\//, "")
+        filename = filename.gsub(/^#{::SimpleCov.root}/, ".").gsub(%r{^./}, "")
```

Comment:
The `.` probably still needs to be escaped, if it's meant to be a literal `.` and not the regex wildcard